File: TsogosunProfileAdmin/MSProfileAdmin/Model/Dtos/StrapiContent/LeaderBoardStrapi.cs
using System;
using tsogosun.com.MSProfileAdmin.Model.Dtos.StrapiContent.LeaderBoardImages;

namespace tsogosun.com.MSProfileAdmin.Model.Dtos.StrapiContent
{
    public class LeaderBoardStrapi
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public string Order { get; set; }
        public string Tier { get; set; }
        public DateTime? Created_At { get; set; }
        public DateTime? Updated_At { get; set; }
        public Image Image { get; set; }
    }
}

File: TsogosunProfileAdminWeb/src/app/model/Dtos/profile-admin/strapiContent/leaderBoardStrapi.ts
import { ImageDetailsDto } from "./leaderBoardImages/imageDetailsDto";

export interface LeaderBoardStrapi {
    id: number;
    title?: string | undefined;
    startDate?: Date | undefined;
    endDate?: Date | undefined;
    order?: string | undefined;
    tier?: string | undefined;
    created_At?: Date | undefined;
    updated_At?: Date | undefined;
    image?: ImageDetailsDto | undefined;
}